documentation/anonymization.markdown

```diff
@@ -1,10 +1,11 @@
 ---
 title: "PuppetDB 4.0: Data anonymization"
 layout: default
-canonical: "/puppetdb/latest/anonymization.html"
 ---
 
-The export tool has the ability to anonymize the PuppetDB archive before returning the archive. This is particularly useful when sharing PuppetDB data that contains sensitive items.
+The export tool has the ability to anonymize the PuppetDB archive before
```

Comment:
So this paragraph right here, and the title, is specific to data anonymization. At least its heavily directed that way.

However, in the new cli client tools docs, we link to it talking mainly about export/import ...

This doc by linking to it this way is becoming something that covers the export/import tools more generally, and anonymization now becomes an option one can use. I'd say change the title of the doc, maybe to 'Export, Import and Anonymization' or something like that, and perhaps have a better intro here.